src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -76,18 +76,19 @@ public void RunAndBlock(CancellationToken cancellationToken = default(Cancellati
                 {
                     IsRunning = false;
 
-                    // Create a new host config, but keep the host id from existing one
+                    // Create a new host config, but keep the host id and timeout from existing one
```

Comment:
Old comment; not needed anymore since the timeout is handled in the ScriptHostConfig now.

---
In reply to: [77680192](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r77680192) [](ancestors = 77680192)